tests/Http/PostTest.php

```diff
@@ -1022,7 +1026,7 @@ public function testPostsIndexAsOwner()
     public function testPostsIndexAsOwnerHiddenPosts()
     {
         // Owners should only see accepted posts and their own hidden posts.
-        $ownerId = $this->faker->northstar_id;
+        $ownerId = $this->faker->unique()->northstar_id;
```

Comment:
I would have thought that since this one came first, the other one would never be the same, but the evidence shows otherwise!

Reply:
Yeah, totes! I don't think it starts tracking uniqueness until you use the `unique()` helper though, so when it was choosing a "unique" value later on it didn't take this into account.